src/main/webapp/app/overview/exercise-details/exercise-details-student-actions.component.ts

```diff
@@ -66,7 +66,7 @@ export class ExerciseDetailsStudentActionsComponent {
      * Starting an exercise is not possible in the exam, otherwise see exercise.utils -> isStartExerciseAvailable
      */
     isStartExerciseAvailable(): boolean {
-        return !this.examMode && isStartExerciseAvailable(this.exercise as ProgrammingExercise);
+        return !this.examMode && isStartExerciseAvailable(this.exercise as ProgrammingExercise, this.studentParticipation);
```

Comment:
You should make sure that the participation without `testRun === true` is used here